lib/client.c

```diff
@@ -199,28 +213,44 @@ void get_real_results(SF_STMT * sfstmt) {
         return;
       }
     }
-  }
-  int sleep_time = retry_pattern[retry] * 500;
+
+    int sleep_time = retry_pattern[retry] * 500;
 #ifdef _WIN32
-  Sleep(sleep_time);
```

Comment:
define this as a common function in platform.c? I think same logic being used in http_perform.c as well.